service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplTest.java

```diff
@@ -277,4 +292,139 @@ public void shouldReturnAssessmentWithItemProps() {
         assertThat(retCon2.getPropertyName()).isEqualTo(constraint2.getPropertyName());
         assertThat(retCon2.getPropertyValue()).isEqualTo(constraint2.getPropertyValue());
     }
+
+    @Test
+    public void shouldReturnAssessmentWithStrands() {
```

Comment:
I could have combined all the different properties into one huge unit test, but I thought it would be simpler to test the mapping of each piece of data separately.